tests/unit_tests/test_common_utils.py

```diff
@@ -36,16 +36,17 @@ class TestMakeClusterNameOnCloud:
     @mock.patch('sky.utils.common_utils.get_user_hash')
     def test_make(self, mock_get_user_hash):
         mock_get_user_hash.return_value = MOCKED_USER_HASH
-        assert "lora-ab12" == common_utils.make_cluster_name_on_cloud("lora")
+        assert "lora-ab12cd34" == common_utils.make_cluster_name_on_cloud(
+            "lora")
 
     @mock.patch('sky.utils.common_utils.get_user_hash')
     def test_make_with_hyphen(self, mock_get_user_hash):
         mock_get_user_hash.return_value = MOCKED_USER_HASH
-        assert "seed-1-ab12" == common_utils.make_cluster_name_on_cloud(
+        assert "seed-1-ab12cd34" == common_utils.make_cluster_name_on_cloud(
             "seed-1")
 
     @mock.patch('sky.utils.common_utils.get_user_hash')
     def test_make_with_characters_to_transform(self, mock_get_user_hash):
         mock_get_user_hash.return_value = MOCKED_USER_HASH
-        assert "cuda-11-8-ab12" == common_utils.make_cluster_name_on_cloud(
+        assert "cud-73-ab12cd34" == common_utils.make_cluster_name_on_cloud(
             "Cuda_11.8")
```

Comment:
might worth to add a line for testing longer max-length
```suggestion
        assert "cud-73-ab12cd34" == common_utils.make_cluster_name_on_cloud(
            "Cuda_11.8")
        assert "cuda-11-8-ab12" == common_utils.make_cluster_name_on_cloud(
            "Cuda_11.8", max_length=20)
```